src/fs/ufs/UFSStoreState.cc

```diff
@@ -64,7 +64,7 @@ void
 Fs::Ufs::UFSStoreState::openDone()
 {
     if (closing)
-        debugs(0, DBG_CRITICAL, HERE << "already closing in openDone()!?");
+        debugs(0, DBG_CRITICAL, "already closing in openDone()!?");
```

Comment:
This PR automatically removes HERE, and nothing else should be required.

The script can auto-replace `0` sections with `1`, but `1` would not be right in many cases, and guessing the right section is too difficult for the script.